service-broker/wso2apim/cf/servicebroker/service-broker-api.bal

```diff
@@ -136,13 +136,19 @@ service<http> serviceBroker {
         }
         string serviceEndpointPassword = strings:valueOf(parameters.serviceEndpointPassword);
 
+        if(parameters.apiDefinition == null) {
```

Comment:
Shall we have space in between if and (parameters...) ? WDYT ?

Reply:
A good catch Dilan! Let me fix this. Thanks!